core/pyproject.toml

```diff
@@ -50,6 +51,7 @@ oracle = ["oracledb", "SQLAlchemy"]
 pg = ["psycopg2-binary","SQLAlchemy"]
 redis = ["redis"]
 sqlserver = ["pymssql", "SQLAlchemy"]
+parquet = ["fastparquet"]
```

Comment:
should we change to `write.parquet` or `block-write.parquet`? the db drivers are actually a sub-dependency of those even, so perhaps we settle on a naming convention? If we want to start introducing many blocks, these will grow quickly and will be hard to manage. In fact, we should probably be placing them in a block descriptor json near the block at some point and compiling them here. But the first step can be a naming convention.
Thoughts?